mmdet/models/anchor_heads/anchor_head.py

```diff
@@ -297,17 +302,19 @@ def get_targets(self,
             gt_bboxes_ignore_list = [None for _ in range(num_imgs)]
         if gt_labels_list is None:
             gt_labels_list = [None for _ in range(num_imgs)]
+        results = multi_apply(
+            self._get_targets_single,
+            concat_anchor_list,
+            concat_valid_flag_list,
+            gt_bboxes_list,
+            gt_bboxes_ignore_list,
+            gt_labels_list,
+            img_metas,
+            label_channels=label_channels,
+            unmap_outputs=unmap_outputs)
         (all_labels, all_label_weights, all_bbox_targets, all_bbox_weights,
-         pos_inds_list, neg_inds_list) = multi_apply(
-             self._get_targets_single,
-             concat_anchor_list,
-             concat_valid_flag_list,
-             gt_bboxes_list,
-             gt_bboxes_ignore_list,
-             gt_labels_list,
-             img_metas,
-             label_channels=label_channels,
-             unmap_outputs=unmap_outputs)
+         pos_inds_list, neg_inds_list) = results[:6]
+        rest_results = list(results[6:])  # user-added return values
```

Comment:
An alternative:
```python
# change the return order of self._get_targets_single() and then
pos_inds_list, neg_inds_list, *others = multi_apply(
    self._get_targets_single,
    concat_anchor_list,
    concat_valid_flag_list,
    gt_bboxes_list,
    gt_bboxes_ignore_list,
    gt_labels_list,
    img_metas,
    label_channels=label_channels,
    unmap_outputs=unmap_outputs)
```

Reply:
This is a little risky, since multiple of modules are involved. The modification should be systematical, and I plan to commit another PR after this one.